chapter8.adoc

```diff
@@ -30,8 +69,64 @@ security manager using secure memory.
 When `medbgen` is 1, there are no restrictions. When `medbgen` is 0 but
 `MSDCFG.sdedbgalw` is 1, then the external debug mode may be entered but is
 restricted by `M-mode` software to prevent privileged CSR accesses and memory
```

Comment:
"restricted by `M-mode` software" is a confusing statement. The extension is a HW/SW co-work solution, not pure software thing.

Reply:
agree - added clarification that M-mode software controls the policy via the msdcfg control bit sdedbalw which gates external debug.